tools/pika_exporter/README.md

```diff
@@ -41,136 +43,138 @@ Run prometheus:
+| 36            | rocksdb.total-sst-files-size            | Total size (in bytes) of all SST files.Note: If there are too many files, it may slow down the online query.                                                                                                          |
+| 37            | rocksdb.live-sst-files-size             | Total size in bytes of all SST files belonging to the latest LSM tree.                                                                                                                                                |
+| 47            | rocksdb.block-cache-capacity            | The capacity of the block cache.                                                                                                                                                                                      |
+| 48            | rocksdb.block-cache-usage               | Memory size occupied by entries in the block cache.                                                                                                                                                                   |
+| 49            | rocksdb.block-cache-pinned-usage        | Memory size occupied by pinned entries in the block cache.                                                                                                                                                            |
+| 51            | rocksdb.num-blob-files                  | The number of blob files in the current version.                                                                                                                                                                      |
+| 52            | rocksdb.blob-stats                      | The total and size of all blob files, and the total amount of garbage (in bytes) in blob files in the current version.                                                                                                |
+| 53            | rocksdb.total-blob-file-size            | The total size of all blob files across all versions.                                                                                                                                                                 |
+| 54            | rocksdb.live-blob-file-size             | The total size of all blob files in the current version.                                                                                                                                                              |
+| 55            | rocksdb.cfstats                         | This includes some statistical information about the column family in the database.                                                                                                                                   |
+
```

Comment:
Perhaps you can list a table described in a table in Readme to describe the description of each stat in the Column Family.